packages/components/tooltip/src/tooltip.tsx

```diff
@@ -364,6 +379,10 @@ export const Tooltip = forwardRef<TooltipProps, "div">(
       <>
         {trigger}
 
+        <ui.span id={tooltipContentId} hidden>
```

Comment:
@ayanami77 

Sorry, I realized this after showing you the method, but you shouldn't use the `hidden` attribute solely for controlling the visibility of elements. Instead, you should use `visibility: hidden` in CSS.